src/styles/blocks/mixins.scss

```diff
@@ -0,0 +1,3 @@
+@mixin margins-default() {
+  margin: 0;
+}
```

Comment:
it makes no sense create mixin for one rule, u can just write margin:0; instead @include margins-default;
better to add * {margin:0;} for all elements